### DISCUSSION TITLE
Alt tags on responsive images

### DISCUSSION THREAD

**Discussion by BradDobson2** [spatie/laravel-medialibrary#1955] [Ideas]
Just wondered if there is a reason that `alt` tags aren't included by default on responsive image templates?

I know I could achieve this by publishing the views and modifying them directly to add in the `alt` tag.

The regular image template adds the `alt` tag using the media object name. Is it feasible for this to be replicated in the responsive image template and the responsive image with placeholder templates?

Any input would be appreciated, thank you.

**Comment by freekmurze** [comment]
I'd accept a PR that improves this.

**Comment by A909M** [comment]
I've submitted a PR to resolve this: [#3765 ](https://github.com/spatie/laravel-medialibrary/pull/3765).
